ui/app/components/clients/history.js

```diff
@@ -231,11 +237,23 @@ export default class History extends Component {
   @action
   selectStartMonth(month) {
     this.startMonth = month;
+    // disables months if in the future
+    this.months.indexOf(month) >= this.currentMonth
```

Comment:
It might be more clear to rewrite this so the ternary is on the right hand side of the statement to determine the conditional value.
`this.disabledYear = this.months.indexOf(month) >= this.currentMonth ? this.currentYear : null;`

Reply:
good call! thank you